dapr/clients/grpc/client.py

```diff
@@ -38,7 +38,7 @@
 from opencensus.trace.tracers.base import Tracer   # type: ignore
 
 
-class DaprClient:
+class DaprGrpcClient:
```

Comment:
Why renaming it if it is in another package?

Reply:
Because there is a new DaprClient which inherits from this class. Would be confusing if I named it the same, wouldn't it?

Reply:
You can name it differently when importing it. It is already in another package.

Reply:
Yeah, but we only named it so generically in the first place because we only had the gRPC client and nothing else. Now that we have both gRPC and HTTP, naming them more specifically makes sence.